src/workloads/query/TenMDocCollection_IntId.yml

```diff
@@ -1,109 +1,172 @@
 SchemaVersion: 2018-07-01
 Owner: "@mongodb/query"
-Description: |
+Description: >
   This workload tests performance of IDHACK on queries with int _id.
 
 KeyWords:
 - IDHACK
 
+GlobalDefaults:
+  Database: &Database test
+  Collection: &Collection Collection0
+  MaxPhases: &MaxPhases 10
+
 Clients:
   Default:
     QueryOptions:
+      connectTimeoutMS: 3_600_000  # 1 hour
       maxPoolSize: 2000
-
-GlobalDefaults:
-  Database: &Database test
-  Collection: &Collection Collection0
-  MaxPhases: &MaxPhases 5
+      socketTimeoutMS: 3_600_000  # 1 hour
 
 ActorTemplates:
 - TemplateName: IDHACK
   Config:
-    Name: {^Parameter: {Name: "Name", Default: "IDHackQuery"}}
-    Type: CrudActor
-    Threads: {^Parameter: {Name: "Threads", Default: 1}}
-    Phases:
-      OnlyActiveInPhases:
-        Active: [{^Parameter: {Name: "OnlyActiveInPhase", Default: {unused: "please specify in which phases this actor should be active."}}}]
-        NopInPhasesUpTo: *MaxPhases
-        PhaseConfig:
-          Duration: 5 minutes
-          Database: *Database
-          Collection: *Collection
-          Operations:
-          - OperationName: find
-            OperationCommand:
-              Filter: {_id: {^RandomInt: {min: 0, max: 9999999}}}
-
-Actors:
-# Clear any pre-existing collection state.
-- Name: ClearCollection
-  Type: CrudActor
-  Database: *Database
-  Threads: 1
-  Phases:
-    OnlyActiveInPhases:
-      Active: [0]
-      NopInPhasesUpTo: *MaxPhases
-      PhaseConfig:
-        Repeat: 1
-        Threads: 1
-        Collection: *Collection
+    LoadConfig:
+      Path: TenMDocCollection_Base.tmpl
+      Key: ActorTemplateConfig
+      Parameters:
+        MaxPhases: *MaxPhases
         Operations:
-        - OperationName: drop
-
-- Name: LoadIntIDs
-  Type: Loader
-  Threads: 10
-  Phases:
-    OnlyActiveInPhases:
-      Active: [1]
-      NopInPhasesUpTo: *MaxPhases
-      PhaseConfig:
-        Repeat: 1
-        Database: *Database
-        MultipleThreadsPerCollection: true
-        CollectionCount: 1
-        DocumentCount: 10000000
-        BatchSize: 1000
-        Document:
-          # The multiplier is multiplied by the ActorID, and added to the value.
-          # Since this is the second actor, the ActorIDs increment for each thread starting at 2.
-          # e.g -2000000 + 2 * 1000000 = 0, -1999999 + 2 * 1000000 = 1, etc.
-          # Once complete, the _ids the collection will have be the range 0-9999999.
-          _id: {^Inc: {start: -2000000, multiplier: 1000000}}
-
-- Name: Quiesce
-  Type: QuiesceActor
-  Threads: 1
-  Database: *Database
-  Phases:
-    OnlyActiveInPhases:
-      Active: [2]
-      NopInPhasesUpTo: *MaxPhases
-      PhaseConfig:
-        Repeat: 1
+        - OperationName: find
+          OperationCommand:
+            Filter: {_id: {^RandomInt: {min: 0, max: 9999999}}}
+- TemplateName: IDFieldPointProject
+  Config:
+    LoadConfig:
```

Comment:
I feel like I've lost the plot here a little bit... why did we split this file into two? Do you have a grand plan? Or did we give up on the grand plan and now we should have one file?

Reply:
Ah, the template file is helpful for simplifying some of the common syntax... but an argument could be made that it's no longer necessary.